models/public/background-matting-mobilenetv2/model.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  The "background-matting-mobilenetv2" model is a high-resolution background replacement technique based on
+  background matting (with MobileNetV2 backbone), where an additional frame of the background is
+  captured and used in recovering the alpha matte and the foreground layer. This model is
+  pre-trained in PyTorch* framework and converted to ONNX* format. More details provided in
+  the paper <https://arxiv.org/abs/2012.07810>.
```

Comment:
please align description text with readme.md
you can use this script for it https://github.com/openvinotoolkit/open_model_zoo/tree/master/ci/documentation_updater